tests/integration/targets/digitalocean_floating_ip/tasks/main.yml

```diff
@@ -5,7 +5,7 @@
   when: do_api_key is not defined
```

Comment:
Regarding the target directory names: why do you keep the `digitalocean` prefix? ansible-test won't find these tests when it detect changes to the `floating_ip` module because the module is called `floating_ip`, and not `digitalocean_floating_ip`. Same for the other integration tests.